src/transformers/models/deformable_detr/image_processing_deformable_detr_fast.py

```diff
@@ -312,23 +273,48 @@ class DeformableDetrImageProcessorFast(BaseImageProcessorFast):
             The size `{"height": int, "width" int}` to pad the images to. Must be larger than any image size
             provided for preprocessing. If `pad_size` is not provided, images will be padded to the largest
             height and width in the batch.
-    """
-
+    """,
+)
+class DeformableDetrImageProcessorFast(BaseImageProcessorFast):
+    resample = PILImageResampling.BILINEAR
+    image_mean = IMAGENET_DEFAULT_MEAN
+    image_std = IMAGENET_DEFAULT_STD
+    format = AnnotationFormat.COCO_DETECTION
+    do_resize = True
+    do_rescale = True
+    do_normalize = True
+    do_pad = True
+    size = {"shortest_edge": 800, "longest_edge": 1333}
+    default_to_square = False
     model_input_names = ["pixel_values", "pixel_mask"]
+    valid_extra_kwargs = [
+        "format",
+        "annotations",
+        "do_convert_annotations",
+        "do_pad",
+        "pad_size",
+        "return_segmentation_masks",
+        "masks_path",
+    ]
 
     def __init__(
         self,
```

Comment:
here and everywhere else, the default fast argument should and can be just ImageKwargs typed no? and not 100% sure we need the **kwargs in child classes!